handlers/whatsapp/whatsapp.go

```diff
@@ -593,25 +658,106 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 		} else {
 			parts := handlers.SplitMsgByChannel(msg.Channel(), msg.Text(), maxMsgLength)
 			externalID := ""
-			for i, part := range parts {
-				payload := mtTextPayload{
-					To:   msg.URN().Path(),
-					Type: "text",
-				}
-				payload.Text.Body = part
-				wppID, externalID, logs, err = sendWhatsAppMsg(msg, sendPath, payload)
 
-				// add logs to our status
-				for _, log := range logs {
-					status.AddLog(log)
+			qrs := msg.QuickReplies()
+
+			channel := msg.Channel()
+
+			wppVersion := channel.ConfigForKey("version", "0").(string)
+			hasCompatibleVersion := wppVersion >= interactiveMsgMinSupVersion
```

Comment:
This is not going to work for string versions. You'll need to use something like the semantic versioning package `semver`

Reply:
Also we need the RapidPro side of this as well right?

Reply:
Hello, nicpottier. To improve version checking i used a semver package in the new commit as you recommended. Does it look better that way?